websites/F/Flat/presence.ts

```diff
@@ -0,0 +1,82 @@
+const presence = new Presence({
+	clientId: "1021848171946704906",
+});
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "https://i.imgur.com/pFfJvOZ.png",
+		},
+		{ pathname } = location;
+	if (pathname === "/") presenceData.details = "Viewing the homepage";
+	if (pathname === "/my-library")
+		presenceData.details = "Viewing their library";
+	if (pathname.startsWith("/collection/")) {
+		presenceData.details = "Viewing a collection:";
+		presenceData.state =
+			document.querySelector(".collection-title")?.textContent;
+	}
+	if (pathname === "/shared-with-me")
+		presenceData.details = "Viewing shared scores";
+	if (pathname === "/trash")
+		presenceData.details = "Viewing recently deleted scores";
+	if (pathname.startsWith("/help/")) {
+		if (pathname.includes("/support"))
+			presenceData.details = "Contacting support";
+		else {
+			const title = document.querySelector(".page-title > h1")?.textContent;
+			presenceData.details = title ? "Viewing help:" : "Viewing help";
```

Comment:
Ideally shouldn't have colons (`:`)
```suggestion
			presenceData.details = title ? "Viewing help" : "Viewing help";
```